proto/kedge/config/grpc/routes/adhoc.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package kedge.config.grpc.routes;
+
+
+/// Adhoc describes an adhoc proxying method that is not backed by a backend, but dials a "free form" DNS record.
+message Adhoc {
+    /// authority_matcher matches the :authority header that will be resolved using A or SRV records.
```

Comment:
`using A or SRV records` this is not true. I think we use `A` only now, right? (:

Reply:
Excellent point, I think we should update the comments in the HTTP adhoc definition to reflect this also.